<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editor</class>
 <widget class="QDialog" name="editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(42, 42, 42);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Consolas&quot;;
color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>Create / Edit Project</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 255);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Project Name :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="projectName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 255);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Project Description :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="projectDescription">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>641</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 255);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Project Tags :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="Tag1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="Tag2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="Tag3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 255);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Github Link:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="githubLink">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 255);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>Executable Path :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="exePath">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 127);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseExe">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>360</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Downloads/kisspng-purple-circle-font-apps-run-executable-5ab0fd635d32b0.8411748115215486433818.png</normaloff>../../../../../Downloads/kisspng-purple-circle-font-apps-run-executable-5ab0fd635d32b0.8411748115215486433818.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
